### PR TITLE
[IMPROVEMENT] Added Data Model for BlockedUsers

### PR DESCRIPTION
**Pull request by TheClashster**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [] I have never used the project.
- [x] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

Includes their userID so that it works even when they change their username.

### CODE REVIEW

File: mod_ci/models.py [hunk 1]
```diff
@@ -15,6 +15,16 @@
...
 
 
+class BlockedUsers(Base):
+    __tablename__ = 'BlockedUsers'
+    # userID refers to the ID from https://api.github.com/users/your_username
+    userID = Column(Integer, primary_key=True)
+
+    def __repr__(self):
+        return "<BlockedUsers(userID='%s')>" % (
+            self.userID)
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: Please use string.format instead

Comment (TheClashster) [reply mod_ci/models.py]: Done 😄

File: mod_ci/models.py [hunk 2]
```diff
@@ -15,6 +15,16 @@
 from mod_test.models import Test, TestPlatform
 
 
+class BlockedUsers(Base):
+    __tablename__ = 'BlockedUsers'
+    # userID refers to the ID from https://api.github.com/users/your_username
+    userID = Column(Integer, primary_key=True)
+
+    def __repr__(self):
+        return "<BlockedUsers(userID='{}')>".format(
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: That can be done on a single line. Also, I'd prefer an named argument for the formatting, it'll be a bit more clear.

File: mod_ci/models.py [hunk 3]
```diff
@@ -15,6 +15,17 @@
 from mod_test.models import Test, TestPlatform
 
 
+class BlockedUsers(Base):
+    __tablename__ = 'BlockedUsers'
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: Please change this to be in line with other table names.

File: mod_ci/models.py [hunk 4]
```diff
@@ -15,6 +15,17 @@
 from mod_test.models import Test, TestPlatform
 
 
+class BlockedUsers(Base):
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: Mysql engine is missing

File: mod_ci/models.py [hunk 5]
```diff
@@ -15,6 +15,17 @@
...
+    __tablename__ = 'BlockedUsers'
+    # userID refers to the ID from https://api.github.com/users/your_username
+    userID = Column(Integer, primary_key=True)
+    comment = Column(String)
+
+    def __repr__(self):
+        return "<BlockedUsers(userID='{TheirID}', comment='{Info}')>".format(
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: please use lowercase for the named arguments.

Comment (TheClashster) [reply mod_ci/models.py]: Sorry, a moment please.

**Pull request merged**